src/components/Dialog/Dialog.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { render, cleanup, fireEvent } from '@testing-library/react';
+import Dialog from './index';
+
+// react
```

Comment:
not sure what this line does

Reply:
You mean cleanup? I want to prevent memory leak and tests which are not "idempotent" (which can lead to difficult to debug errors in your tests), that's why I call cleanup after each test.